components/maintenance/lead_acid_battery_care.mdx

```diff
@@ -1,19 +1,19 @@
-Husky's default power supply is a sealed 24 V lead-acid battery pack (VRLA), providing 20 Ah of charge.
-These tips are intended to help keep your Husky Lead-Acid Battery in tip-top shape.
-With proper maintenance, the battery should maintain the majority of its capacity for hundreds of cycles.
-
 The most damaging thing to a lead-acid battery is a phenomenon called sulfation.
 When a lead-acid battery is left in an uncharged state for long periods of time, sulfate
 crystals solidify on the electrodes of the battery.
 This effect can be permanent, and it causes premature reduction in capacity.
 Therefore, it is important to fully charge the battery as soon as you are finished using it,
 regardless of how much capacity remains.
 
-Your Husky ships with a three-stage battery charger that completely charges a Husky battery.
-Charging occurs in three stages: Constant current, topping charge, and float charge.
+Your robot's battery charger works in 3 stages:
```

Comment:
I think that Husky now has a 2-stage charger. I only know this since JB recently had me make a change to the User Manual. Not sure that will affect this common section.